build.gradle

```diff
@@ -249,6 +249,7 @@ dependencies {
   implementation group: 'org.springframework.security', name: 'spring-security-crypto', version: '5.7.5'
 
   implementation group: 'com.fasterxml.jackson.core', name:  'jackson-databind', version: '2.14.0'
+  implementation group: 'commons-fileupload', name: 'commons-fileupload', version: '1.5'
```

Comment:
May be it should be tested on perftest first to confirm it doesn't "break" anything